src/scripts/config.js

```diff
@@ -319,6 +317,14 @@ async function processModule(filePath, cachedIds, registry, ids, envs) {
     }
   }
 
+  if (!isJson) {
+    const transformed = await esbuild.transform(code, {
+      format: 'cjs',
+      target: 'es2015'
+    });
+    code = transformed.code;
+  }
+
```

Comment:
not sure if this is where this should go, but was getting `Unexpected "export"` errors